packages/magellan-mocha-plugin/test/get_suites.spec.js

```diff
@@ -1,39 +1,40 @@
 const path = require( 'path' );
-const chai = require( 'chai' );
-const expect = chai.expect;
 const Locator = require( '../lib/locator' );
 const testFramework = require( '../index' );
+const fs = require( 'fs' ).promises;
+const os = require( 'os' );
 
-function getTestsFrom( specs ) {
+async function getTestsFrom( specs ) {
 	if ( ! Array.isArray( specs ) ) {
 		specs = [ specs ];
 	}
 	testFramework.initialize( {
 		mocha_tests: specs,
-		mocha_opts: path.join( specs[ 0 ], 'mocha.opts' ),
+		mocha_config: path.join( specs[ 0 ], '.mocharc.js' ),
 		suiteTag: 'suite;multiple',
 	} );
-	return testFramework.iterator( { tempDir: path.resolve( '.' ) } );
+	const tempDir = await fs.mkdtemp( path.join( os.tmpdir(), 'magellan-mocha-plugin' ) );
```

Comment:
`testFramework.initialize()` will write a JSON file to the filesystem and then delete it (🤦🏼 ). 
Because tests in Jest may run in parallel, we need to ensure the path is unique.